From phil@ipom.com Sat Feb 24 19:24:15 2007
From: Phil Dibowitz <phil@ipom.com>
Date: Sat, 24 Feb 2007 19:23:41 -0800
Subject: USB storage: Removed duplicate supertop unusual_dev entry
To: Phil Dibowitz <phil@ipom.com>
Cc: Greg KH <gregkh@suse.de>, <thomas@archlinux.org>, <r.schedel@yahoo.de>
Message-ID: <45E1013D.4010602@ipom.com>

This patch removes the duplicate supertop entries that made it into the
.21 rc kernels.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |   10 ----------
 1 file changed, 10 deletions(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -1395,16 +1395,6 @@ UNUSUAL_DEV(  0x1652, 0x6600, 0x0201, 0x
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_RESIDUE ),
 
-/* Reported by Thomas Baechler <thomas@archlinux.org>
- * Fixes I/O errors with Teac HD-35PU devices
- */
-
-UNUSUAL_DEV( 0x1652, 0x6600, 0x0201, 0x0201,
-               "Super Top",
-               "USB 2.0  IDE DEVICE",
-               US_SC_DEVICE, US_PR_DEVICE, NULL,
-               US_FL_IGNORE_RESIDUE),
-
 /* patch submitted by Davide Perini <perini.davide@dpsoftware.org>
  * and Renato Perini <rperini@email.it>
  */
